src/components/Tabs/Tabs.tsx

```diff
@@ -113,9 +118,12 @@ class TabsInner extends React.PureComponent<CombinedProps, State> {
       disclosureActivatorVisible && styles.fillSpace,
     );
 
+    const activatorClassName = classNames(styles.Title, styles.activator);
+
     const disclosureTabClassName = classNames(
       styles.DisclosureTab,
       disclosureActivatorVisible && styles['DisclosureTab-visible'],
+      newDesignLanguage && styles['DisclosureTab-newDesignLanguage'],
```

Comment:
Note:

Would prefer this to be,

```
newDesignLanguage & styles.newDesignLanguage`
```

and have a `DisclosureTab` class within `newDesignLanguage`.

Reply:
me too :) but we haven't been consistent.